libtiledbvcf/src/stats/variant_stats_reader.cc

```diff
@@ -171,21 +404,30 @@ void VariantStatsReader::wait() {
   }
 }
 
-std::pair<bool, float> VariantStatsReader::pass(
+std::tuple<bool, float, uint32_t, uint32_t> VariantStatsReader::pass(
     uint32_t pos,
     const std::string& allele,
     bool scan_all_samples,
     size_t num_samples) {
+  if (array_version() > 2) {
+    af_map_.advance_to_ref_block(pos);
+  }
   if (!allele.compare("<NON_REF>")) {
-    return {false, 0.0};
+    // TODO: replace placeholder return values if necessary
```

Comment:
Is this `TODO` done?

Reply:
I'd wait until the following release to ensure that we don't want to reexamine that. From a conversation earlier with @leipzig, he told me to leave it for now.